src/components/Choropleth.vue

```diff
@@ -129,29 +131,43 @@ const style = (f: Feature) => {
 };
 
 const updateBounds = async () => {
-    // update bounds
+    const rawMap = toRaw(map.value);
```

Comment:
`toRaw` unpacks an object and returns it without vue reactivity attached to it, we are not using the map in a reactive way (to trigger changes for example) so this should be fine, i think this gets rid of the map is null errors that @EmmaLRussell showed here: https://github.com/mrc-ide/arbomap/pull/13#pullrequestreview-2035690703

Reply:
i found the original link (also put that in the comment) and have tried to explain it in my own words in terms of what i think is happening with the vue reactivity!

Reply:
in putting the explanation online into my own words i realised that the `shallowRef` stuff is sufficient actually! basically, vue 3 deep watches refs by default but that means that it updates every time the leaflet map object updates which has its own reactivity built into it. So we lose the reference to `map` sometimes when we try and do an update to `map` because vue 3 wants to react to the changes to the internals of `map`